protocol/communities_messenger_test.go

```diff
@@ -495,6 +495,42 @@ func (s *MessengerCommunitiesSuite) TestPostToCommunityChat() {
 	s.Require().True(found)
 }
 
+func (s *MessengerCommunitiesSuite) TestPinMessageInCommunityChat() {
+	community, chat := s.createCommunity()
+	s.Require().NotNil(community)
+	s.Require().NotNil(chat)
+	community.AdminSettings().PinMessageAllMembersEnabled = true // FIXME: this doesn't work
+	fmt.Println("AAAAAAA", community.AllowsAllMembersToPinMessage())
+	err := s.admin.communitiesManager.SaveCommunity(community)
+	fmt.Println("BBBBBBB", community.AllowsAllMembersToPinMessage())
+	s.Require().NoError(err)
+
+	s.advertiseCommunityTo(community, s.alice)
+	s.joinCommunity(community, s.alice)
+
+	inputMessage := common.NewMessage()
+	inputMessage.ChatId = chat.ID
+	inputMessage.ContentType = protobuf.ChatMessage_TEXT_PLAIN
+	inputMessage.Text = "pinned message"
+
+	ctx := context.Background()
+	sendResponse, err := s.alice.SendChatMessage(ctx, inputMessage)
+	s.Require().NoError(err)
+	s.Require().Len(sendResponse.Messages(), 1)
+
+	community, err = s.admin.communitiesManager.GetByID(community.ID())
```

Comment:
Grabbing the community again, either via s.admin or via s.[alice|bob] doesn't not yield the CommunityAdminSettings member which was set initially at community creation.